felles/src/main/kotlin/no/nav/familie/kontrakter/felles/personopplysning/Personinfo.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.kontrakter.felles.personopplysning
+
+import no.nav.familie.kontrakter.felles.personopplysning.adresse.Adresseinfo
+import no.nav.familie.kontrakter.felles.personopplysning.relasjon.Familierelasjon
+import no.nav.familie.kontrakter.felles.personopplysning.relasjon.SivilstandType
+import no.nav.familie.kontrakter.felles.personopplysning.status.PersonstatusType
+import no.nav.familie.kontrakter.felles.personopplysning.tilhørighet.Landkode
+import java.time.LocalDate
+import java.util.*
+
+class Personinfo(val personIdent: PersonIdent,
```

Comment:
Gir det mening å trekke ut dette? Vanskelig å minimere data når kontrakten her inneholder alt. @hensol hva tenker du om dette? 

Akkurat rundt pdl og personinfo kan det kanskje være lurt å skille mellom ytelser? Med mindre vi trenger akkurat det samme såklart! :)

Reply:
Veldig gode poenger, og de samme poengene jeg selv kom med når vi gikk for å lage felles integrasjoner. Jeg gikk nok litt på autopilot når jeg lagde dette. Foreslår at vi skroter denne branchen. De enkelte løsningene burde ikke hente ut mer enn akurat det de trenger.

Så lenge vi bruker kotlin, så blir jo uansett disse dto-ene veldig lite arbeid å lage.